packages/packages/top-bar/src/components/locations/main-menu-location.tsx

```diff
@@ -16,7 +16,7 @@ export default function MainMenuLocation() {
 	} );
 
 	return (
-		<Stack sx={ { px: 4 } } direction="row" alignItems="center">
+		<Stack sx={ { paddingInlineStart: 4 } } direction="row" alignItems="center">
```

Comment:
why? there is no need for padding at the end?

Reply:
it comes from the tools section which has the same element as the utilities section and both of them needs the padding from the sides..